libs/application/templates/parental-leave/src/fields/Rights/RequestRights.tsx

```diff
@@ -54,20 +64,35 @@ const RequestRights: FC<FieldBaseProps> = ({ error, field, application }) => {
           <Text color="red400">{formatMessage(mm.errors.requiredAnswer)}</Text>
         </Box>
       )}
-      <BoxChart
-        application={application}
-        boxes={numberOfBoxes}
-        calculateBoxStyle={(index) => {
-          if (index === 6) {
-            if (statefulAnswer === 'yes') {
-              return 'greenWithLines'
+
+      {/* No answer yet, so show them the last box as gray */}
+      {!statefulAnswer && (
+        <BoxChart
+          application={application}
+          boxes={numberOfBoxes}
+          calculateBoxStyle={(index) => {
+            if (index === 6) {
+              return 'grayWithLines'
             }
-            return 'grayWithLines'
-          }
-          return 'blue'
-        }}
-        keys={boxChartKeys as BoxChartKey[]}
-      />
+            return 'blue'
+          }}
+          keys={boxChartKeys as BoxChartKey[]}
+        />
+      )}
+
+      {/* They won't request time, so show all as blue */}
+      {statefulAnswer === 'no' && (
```

Comment:
I saw there is a constant file containing the YES and NO variables, I would probably use it all over the places to depends on it

Reply:
Good point, should be updated now.